packages/aws-cdk-lib/aws-cognito/test/user-pool.test.ts

```diff
@@ -293,13 +293,16 @@ describe('User Pool', () => {
 
   test('import using arn', () => {
     // GIVEN
-    const stack = new Stack();
+    const stack = new Stack(undefined, undefined, {
+      env: { region: 'us-east-1', account: '0123456789012' },
+    });
```

Comment:
Can we keep the stack identical for this test, or does it need these additional properties to work? If so, that would probably make these breaking changes. If not, can we update the logic that checks for the new `userPoolProviderName` so that it works without additional props?